server/services/__tests__/offenderService.spec.js

```diff
@@ -205,14 +206,54 @@ describe('Offender Service', () => {
       });
 
       const data = await service.getEventsFor(
-        TEST_BOOKING_ID,
+        TEST_USER,
```

Comment:
this test was always failing for the wrong reason as a number was being passed instead of a test user.